crucible-llvm-syntax/crucible-llvm-syntax.cabal

```diff
@@ -0,0 +1,124 @@
+Cabal-version: 2.2
+Name:          crucible-llvm-syntax
+Version:       0.1
+Author:        Galois Inc.
+Maintainer:    langston@galois.com
+Build-type:    Simple
+License:       BSD-3-Clause
+License-file:  LICENSE
+Category:      Language
+Synopsis:      A syntax for reading and writing Crucible-LLVM control-flow graphs
+-- Description:
+
+extra-doc-files: README.md
+extra-source-files:
```

Comment:
Your golden test files should be listed here to ensure they are included in any sdist package.

Reply:
Nice catch!

Reply:
We probably have this issue a lot of places.   We're not sdist-ing these ATM, so it hasn't been a problem, but yes, we should clean this up wherever we can.  It will probably manifest other than sdist at times: I recently discovered that if your cabal.project references a local package path that you will be fine but if it references a source-repository then you will encounter this issue as well.

Reply:
Indeed, I have used `source-repository-package`s instead of `git` submodules on occasion for things under the `GaloisInc` repo, and this issue has bitten me every now and then. It would be good to be consistent about including things in `extra-source-files` going forward, if nothing else.